templates/tfengine/components/devops/variables.tf

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{$missing_admins_group := not (get .admins_group "exists")}}
+variable "admins_group" {
+  type = object({
+    {{- if $missing_admins_group}}
```

Comment:
also reorganize here as in the terraform.tfvars

Only id is needed in both cases. And all other attributes are conditional on missing_admins_group being true.

Reply:
I'm sorry I made a mess on this PR 😣. I got confused at many points.

Fixed